docs/reference/builder.md

```diff
@@ -1470,6 +1474,116 @@ The `STOPSIGNAL` instruction sets the system call signal that will be sent to th
 This signal can be a valid unsigned number that matches a position in the kernel's syscall table, for instance 9,
 or a signal name in the format SIGNAME, for instance SIGKILL.
 
+## SHELL
+
+	SHELL executable parameters
```

Comment:
Do you think this example ought to be JSON form too? (ie, `SHELL ["executable", "parameter", "parameter"]` or something)